packages/0x.js/src/contract_wrappers/generated/ether_token.ts

```diff
@@ -0,0 +1,363 @@
+/**
+ * This file is auto-generated using @0xproject/typed-contracts. Don't edit directly.
+ * Templates can be found at https://github.com/0xProject/0x.js/tree/development/packages/typed-contracts-templates.
+ */
+import {BigNumber} from 'bignumber.js';
+import * as Web3 from 'web3';
+
+import {TxData, TxDataPayable} from '../../types';
+import {classUtils} from '../../utils/class_utils';
+import {promisify} from '../../utils/promisify';
+
+import {BaseContract} from './base_contract';
+
+export class EtherTokenContract extends BaseContract {
+    public name = {
+        async callAsync(
+            defaultBlock?: Web3.BlockParam,
+        ): Promise<string
+    > {
```

Comment:
Is this a limitation of the template language or could we remove some of these superfluous new lines? It makes the code harder to read.

Reply:
I wasn't able to find where to disable this behavior.
I've tried disableIndent compiler option, whitespace control and removing trailing newline at template files.
Those files are auto-generated and shouldn't be read by people very often. They just act like typings. The fact that they have an almost perfect indentation is a nice bonus. The fact that they lint correctly is even nicer. The choosen template language mainly focuses on html. This has it's pros - it's simple. But also has it's cons. it's too simple ;) I'll leave it as a wontfix.